### PR TITLE
Increased python version to 3.7

### PR DESCRIPTION
**Pull request by nabobalis**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge/aioftp This is ready for review and merge!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,3 +36,5 @@ about:
 extra:
   recipe-maintainers:
     - epruesse
+    - nabobalis
+    - cadair
```
Comment (Cadair) [inline recipe/meta.yaml:40]: :+1: I am also happy to help out here as it's a dep of the parfive feedstock :)

**Comment by mbargull** [comment]
I'm fairly certain Elmar won't mind the helping hands :).
(If not, I'll gladly take a "digital slap" from @epruesse for merging on this behalf ;).)

**Comment by nabobalis** [comment]
Thank you @mbargull

**Pull request merged**